client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue

```diff
@@ -58,6 +58,7 @@
           @save="(data) => save(data)" />
 
         <statement-publication-and-voting
+          v-if="hasPermission('feature_statements_vote')"
```

Comment:
this one is a bit tricky. We already use the permission inside the component for a part of what happens there. 
The publication should be displayed even if the permission is off.
client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementPublicationAndVoting.vue:31

Reply:
Maybe, we need another permission like "feature_statement_publication"

Reply:
Maybe we can remove the Permission inside the Component and use that in StatementMeta Component, because it shows user this part anyway
`<fieldset data-dp-validate="statementPublicationAndVotingData">
    <legend
      id="publicationAndVoting"
      class="mb-3 color-text-muted font-normal">
      {{ Translator.trans('publication.and.voting') }}
    </legend>
    <div class="font-semibold">
      {{ Translator.trans('publish.on.platform') }}
    </div>`
    before to check permission. Or we can use Permission at the top of the StatementPublicationAndVoting Component in main template

Reply:
Because I think create a new Permission for this thing is not useful when we have already one for it

Reply:
but the votes and the publication are two different sections as far as I can see. And the permission is already in use. So we can't just move it the break the logic in some projects.